packages/components/tests/integration/components/hds/table/index-test.js

```diff
@@ -243,6 +261,66 @@ module('Integration | Component | hds/table/index', function (hooks) {
       .hasText('Sorted by artist descending');
   });
 
+  test('it renders a custom sortedMessageText if supplied', async function (assert) {
+    setSortableTableData(this);
+    this.set('sortedMessageText', 'Melanie will sort it');
+
+    await render(hbsSortableTable);
+    await click('#data-test-table .hds-table__th-sort:nth-of-type(1) button');
+    assert.dom('#data-test-table caption').hasText('Melanie will sort it');
+
+    await click('#data-test-table .hds-table__th-sort:nth-of-type(1) button');
+    assert.dom('#data-test-table caption').hasText('Melanie will sort it');
```

Comment:
@MelSumner setting the `sortedMessageText` this way, makes the argument always true, so it's always rendered no matter what. We can test it only once (unless we want to test the relation of the message with the sorting and sorting order, in which case we have to change the way we build the message and make it dependant on the table state).

```suggestion
    await render(hbsSortableTable);
    assert.dom('#data-test-table caption').hasText('Melanie will sort it');
```